crates/libcontainer/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "libcontainer"
-version = "0.0.1"
+rust-version = "1.58.1"
+version = "0.0.2"
```

Comment:
Could you have all crates set to 0.0.2?

Reply:
Sure, I only set these two as I had added the rust-version field to these two, but I can add those, as well as fix some inconsistent styling and whitespace issues

Reply:
Hey I have updated all crate's version numbers to 0.0.2, please take a look now.